bigquery_etl/cli/query.py

```diff
@@ -1217,7 +1217,7 @@ def schema():
     default="analysis",
 )
 @use_cloud_function_option
-@respect_dryrun_skip_option(default=True)
+@respect_dryrun_skip_option(default=False)
```

Comment:
If we keep `True` then things like `baseline_clients_first_seen_v1` will be skipped